src/Model/TrialOnlyPlanTrial.php

```diff
@@ -15,7 +15,7 @@
 
 use JsonSerializable;
 
-class PlanTrial implements JsonSerializable
+class TrialOnlyPlanTrial implements JsonSerializable
```

Comment:
I wonder why this renamed.

Reply:
This schema had a generator level workaround broken by this line https://github.com/Rebilly/api-definitions/pull/1651/files#diff-cd46e82f3d934fd5d06b914430717553bbaf2bb9ddf7a55fb19b7f1a9b67a5e6R14
Would address separately after schema flattening